app/react-native/src/server/config/defaults/webpack.config.js

```diff
@@ -1,63 +1,7 @@
-import autoprefixer from 'autoprefixer';
```

Comment:
`loaders` field is deprecated in webpack. Technically, it's a breaking change, but I can't really imagine why would someone use a custom webpack config in RN storybook. It doesn't apply to stories themselves anyway.

Generally, the differences are there because changes to configs in react and vue weren't applied here

Reply:
I guess we need to factor our the RN manager config into `@storybook/core` too. Maybe given the above it is easier than you thought @igor-dv?